File: routeMap/app/src/main/java/ru/android/zheka/model/HomeModel.java
package ru.android.zheka.model;

import android.content.Context;

import androidx.databinding.ObservableField;
import ru.android.zheka.core.AktionMessage;
import ru.android.zheka.core.Message;
import ru.android.zheka.coreUI.ButtonHandler;
import ru.android.zheka.coreUI.PanelModel;

public class HomeModel extends PanelModel implements IHomeModel {
    private ObservableField <ButtonHandler> startButton;


    public HomeModel(Context view) {
        super (view);
    }

//    @Override
//    public void setStartButton(ObservableField button) {
//        this.startButton = button;
//    }
//
//    @Override
//    public ObservableField <ButtonHandler> getStartButton() {
//        return this.startButton;
//    }

    @Override
    public String getInput() {
        return null;
    }

    @Override
    public AktionMessage getAction() {
        return null;
    }

    @Override
    public Message getMessage() {
        return null;
    }
}

File: routeMap/app/src/main/java/ru/android/zheka/model/HomeModel.kt
package ru.android.zheka.model

import android.content.Context
import ru.android.zheka.core.AktionMessage
import ru.android.zheka.core.Message
import ru.android.zheka.coreUI.PanelModel

class HomeModel(view: Context?) : PanelModel(view), IHomeModel {
    override val input: String?
        get() = TODO("Not yet implemented")
    override val action: AktionMessage?
        get() = TODO("Not yet implemented")
    override val message: Message?
        get() = TODO("Not yet implemented")
}